features/organized-chaos.js

```diff
@@ -0,0 +1,6 @@
+setTimeout(() => {
+  javascript:(function(){['', '-ms-', '-webkit-', '-o-', '-moz-'].map(function(prefix){document.body.style[prefix + 'transform'] = 'rotate(180deg)';});}());
```

Comment:
This has been bugging me. Why is there a JavaScript URI inside a JavaScript file?

Reply:
Wtf even is that feature

Reply:
> Wtf even is that feature

Oh wait this pr is for that feature 💀

Reply:
is that not a syntax error

Reply:
I think I'll regret testing this if it does work...

Reply:
> is that not a syntax error

It's technically not. I'm assuming this code was found somewhere else.

Reply:
the code is from my former cs teacher. it's not mine.